src/classwork/04_assign/main.cpp

```diff
@@ -10,6 +11,19 @@ factorial.  Also, loop continues as long as user wants to.
 */
 int main() 
 {
-
+	int choice;
+	
+	
+	do
+	{
+		int num;
+		cout <<"Enter a number between 1 and 10 to display it's factorial: ";
+		cin >>num;
+		cout <<factorial(num)<<"\n\n";
```

Comment:
Missing validation after line 21 for a number from 1 to 10.
Something like this:

               while(num < 1 || num > 10)
		{
			cout<<"Number out of range, enter num from 1 to 10: ";
			cin>>num;
		}